config/custom_items.txt

```diff
@@ -477,6 +477,11 @@ item_path: /obj/item/weapon/implanter/reagent_generator/savannah
 }
 
 # ######## M CKEYS
+{ckey: mewchild
```

Comment:
Line break here please, for consistency with the other entries.

Reply:
Can't believe I missed that. Thank you!